src/modules/utility-service/UtilityServiceController.ts

```diff
@@ -1,12 +1,121 @@
 import { Router } from "express";
+import proxy = require("express-http-proxy");
 import { Controller } from "../../interfaces/Controller";
+import { UTILITY_ROUTES } from "../../routes";
+
+const UTILITY_API = process.env.UTILITY_URI || 'localhost:9000';
 
 export class UtilityServiceController implements Controller {
     buildRouter(): Router {
         const router = Router();
+        
+        /**
+         * @swagger
+         * /status:
+         *  get:
+         *      description: Gets the status for the banner
+         *      tags:
+         *          - Utility Service
+         *      responses:
+         *          200:
+         *              description: OK - Returns status report
+         *              content:
+         *                  application/json:
+         *                          schema:
+         *                              $ref: '#/components/schemas/Status'
+         */
+        router.get(
+            '/status',
+            this.proxyRequest((req: Request) => '/status')
+        );
 
-        // Routes go here
+        /**
+         * @swagger
+         * /maintenance:
+         *  get:
+         *      description: Gets the maintenance status for maintenance page
+         *      tags:
+         *          - Utility Service
+         *      responses:
+         *          200:
+         *              description: OK - Returns the maintenance status as a boolean from the mongodb downtime collection
+         *                      
+         *                  
+         */
+        router.get(
+            `/maintenance`,
+            this.proxyRequest((req: Request) => '/maintenance')
+        );
 
+         /**
+         * @swagger
+         * /clientversion/{clientVersion}:
+         *  get:
+         *      description: Gets the client version to see if there is an update
+         *      tags:
+         *          - Utility Service
+         *      parameters:
+         *          - in: path
+         *            name: clientVersion
+         *            schema:
+         *                type: string
+         *            required: true
+         *            description: The version of the client
+         *      responses:
+         *          200:
+         *              description: OK
+         *          401:
+         *              description: UNAUTHENTICATED - Thrown when no clientVersion is in params
```

Comment:
Quick question is this supposed to be a 401?  This seems more like a 400 error but we may have incorrectly mapped it

Reply:
It was returning a 401, I do agree a 400 seems more appropriate